docs: add no bash flag documentation (#186)
okteto/docs: src/content/reference/cli.mdx

If the repositories in the `dependencies` section haven't been deployed yet, `okteto deploy` deploys these dependencies automatically.

The command is executed relative to the path where the manifest is located. This means, that if the manifest is at any subfolder, the pipeline will use the context of the subfolder where the manifest is defined.

> Executed commands will be wrapped by `bash -c` unless the user specify it by flag.

Reviewer: Instead of wrapped can we say that the commands will be executed using bash?

> Executed commands will be executed using bash unless the `--no-bash` flag has been specified.

Reviewer: I think this might look better as a quote/note like we do for other comment and suggestions

| _--timeout_                                 | (duration) | The duration to wait for the development environment deployment to complete. Any value should contain a corresponding time unit e.g. 1s, 2m, 3h (default 5m0s) |
| _--var_                                     |   (list)   | Set a pipeline variable (can be set more than once)                                                                                                            |
| _--wait_                                    |   (bool)   | Wait until the pipeline finishes (defaults to false)                                                                                                           |

> | _--no-bash_                                 |   (bool)   | Execute commands without wrapping with bash (default to false)                                                                                                 |

Reviewer: Is "wrapping" the right term  for this?

Reviewer: I think we can be more explicit by saying "it will execute the command using the container's default shell instead if bash" ?

### destroy

Destroy your development environment.
It automatically destroys all the Kubernetes resources created by [okteto deploy](/docs/reference/cli/#deploy).
If you need to destroy external resources (like s3 buckets or other Cloud resources), use the [destroy](/docs/reference/manifest/#destroy-string-optional) section of the Okteto Manifest.

> Executed commands will be wrapped by `bash -c` unless the user specify it by flag.

Reviewer 2: ```suggestion
Executed commands will be wrapped by `bash -c` unless the `--no-bash` flag has been specified.
```

| _--timeout_                              | (string) | The duration to wait for the development environment to be destroyed. Any value should contain a corresponding time unit e.g. 1s, 2m, 3h (default 5m0s) |
| _--volumes_                              |  (bool)  | Destroy the persistent volumes created by the development environment (defaults to `false`)                                                             |
| _--wait_                                 |  (bool)  | Wait until the development environment is destroyed (defaults to `false`)                                                                               |

> | _--no-bash_                              |  (bool)  | Execute commands without wrapping with bash (default to false)                                                                                          |

Reviewer 2: ```suggestion
| _--no-bash_                              |  (bool)  | Execute commands without wrapping with bash (defaults to false)                                                                                          |
```
